transitmatters/t-performance-dash
# remove separate pages for single/range

## Motivation

I previously split up `singleday` and `range` pages into separate paths which no longer needs to be done. This will align with upcoming nav rework #458 

## Changes

Delete `pages/[line]/singleday/*` and `pages/[line]/range/*`
Add `pages/[line]/trips/*`

There is currently no way to navigate to these pages in the UI. They highlight the ridership tab when open, but that will be removed in next PR (and was already happening)

## Testing Instructions

Go to paths: `/red/trips/traveltimes` `/red/trips/dwells` `/red/trips/headways` `/red/trips` and verify there is content.
